### DISCUSSION TITLE
Keyboard can work on android mobile devices for text input?

### DISCUSSION THREAD

**Discussion by rogerrey** [ruffle-rs/ruffle#2786] [Q&A]
Hi.
Do you know if it is possible or will it be possible that the keyboard can work on android mobile devices for text input?

**Comment by danielhjacobs** [comment]
This is possible now, specifically for text input.

**Reply by philooo2** [reply]
can the keyboard from a mobile device communicate with the game, in order to interact with the gameplay itself ?
